tap_google_search_console/client.py

```diff
@@ -159,14 +46,16 @@ def _get_resource(self, name: str):
         return getattr(self._service, name)
 
     @backoff.on_exception(backoff.constant,
-                          GoogleForbiddenError,
+                          HttpError,
```

Comment:
do we still need both backoffs?

Reply:
Yes, we do. There are two types of [Load quota](https://developers.google.com/webmaster-tools/search-console-api-original/v3/limits#load-quota) - Short-term and Long-term. If user exceed Short quota, he shoud wait 15 minutes and try again. If user still exceed quota, user is exceeding long-term quota. Long-term quota is measured in 1 day chunks. Therefore we need retry only one time. First backoff handles this case. Another backoff exponentially handels 429 error and server errors with seven retries.

Reply:
I see, thanks for the explanation